services/web/server/src/simcore_service_webserver/director/director_api.py

```diff
@@ -74,6 +76,7 @@ async def start_service(
     headers = {
         "X-Dynamic-Sidecar-Request-DNS": request_dns,
         "X-Dynamic-Sidecar-Request-Scheme": request_scheme,
+        SIMCORE_USER_AGENT_HEADER: request_simcore_user_agent,
```

Comment:
Potential improvement opportunity: hard code all 3 strings here or define all three as constants, but do it the same for all ;)

Reply:
let's go for it